src/scripts/components/stories/story-list-item/story-list-item.styl

```diff
@@ -37,11 +37,8 @@
 
 .info
   overflow-y: auto
-  box-sizing: border-box
```

Comment:
Is there a reason you removed this here? Often times unintuitive stuff happens when this is not set :D

Reply:
Yeah you're right. The percentages also make more sense then.